Documentation/Functions/Stdwrap.rst

```diff
@@ -758,8 +758,7 @@ hash
 :aspect:`Description`
    Returns a hashed value of the current content. Set to one of the
    algorithms which are available in PHP. For a list of supported
-   algorithms see `https://www.php.net/manual/en/function.hash-algos.php
-   <https://www.php.net/manual/en/function.hash-algos.php>`_ .
+   algorithms see https://www.php.net/manual/en/function.hash-algos.php.
```

Comment:
Now it's not linked anymore, is it?

Reply:
Simple URLs are linked by reST / Sphinx automatically unless unlinked explicitly by `:samp:`, see https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Hyperlinks.html#preventing-links.